src/odemis/gui/cont/streams.py

```diff
@@ -3083,3 +3084,124 @@ def _on_overview_streams(self, _):
             if isinstance(s, FastEMOverviewStream) and s not in tab_streams:
                 tab_streams.append(s)
                 canvas.view.addStream(s)
+
+
+class CryoAcquiredStreamsController(StreamBarController):
+    """
+    StreamBarController to control the display of the Static streams in the Cryo
+    Localization tab. It deals with two types of streams:
+     * Overview streams: shown only in the overview view, and always.
```

Comment:
This sentence ends weird, I think I get what you want to say but I can't formulate it any better.